R/carehomes_rt.R

```diff
@@ -28,7 +28,13 @@ carehomes_Rt <- function(step, S, p) {
 
   calculate_ev <- function(t, S, drop_carehomes) {
     ## Next-Generation-Matrix
-    ngm <- beta[t] * outer(mean_duration[, t], S[, t]) * p$m
+    m <- p$m
+    ages <- seq_len(p$N_age - 2)
+    ch <- seq(to = p$N_age, length.out = 2)
+    m[ages, c(ages, ch)] <- beta[t] * m[ages, c(ages, ch)]
```

Comment:
this would be clearer if you omitted c(ages, ch) which is 1:19 i.e. all the columns
```suggestion
    m[ages,  ] <- beta[t] * m[ages,  ]
```